package.json

```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "bcrypt": "^5.0.1",
+    "compression": "^1.7.4",
```

Comment:
There are types for ```compression``` in ```@types/compression```. More information can be found here: https://www.npmjs.com/package/@types/compression. Please install the package for better type integration.

Reply:
PS: please also remember to add it to ```devDependencies``` to prevent installation in production enviroments!

Reply:
Resolved in [d092e8e](https://github.com/Chillandchat/api/pull/16/commits/d092e8e99f3d2013456b17c11e06cad87fb87fbc)